spec/controllers/api/customers_controller_spec.rb

```diff
@@ -3,13 +3,32 @@
 module Api
   describe CustomersController, type: :controller do
     include AuthenticationWorkflow
+    include OpenFoodNetwork::ApiHelper
     render_views
 
     let(:user) { create(:user) }
-    let(:customer) { create(:customer, user: user) }
-    let(:params) { { format: :json, id: customer.id, customer: { code: '123' } } }
+
+    describe "index" do
+      let!(:customer1) { create(:customer) }
+      let!(:customer2) { create(:customer) }
+
+      before do
+        user.customers << customer1
```

Comment:
This modelling reads so weird: "A user has customers." And it doesn't mean that the user is an enterprise. ;-) Maybe a better name for customer would be *customer relationship*, but that's not as quick to write and read... It's probably fine as it is. But I do find it confusing to see a variable called *customer* and it not being a user. It's beyond the scope of this pull request anyway.